### ISSUE TITLE
Переработка метода ПолучитьМассивОписанияВозможныхКонтактов

### ISSUE THREAD

**Issue by pallid** [BlizD/Tasks#126]
добавленный код в метод ПолучитьМассивОписанияВозможныхКонтактов, сделанный коммитом https://github.com/BlizD/Tasks/commit/0b56b96c4adf2d8f5cf4e2b6794bb017b3b77378 необходимо перенести в общий модуль ВзаимодействияКлиентСерверПовтИспПереопределяемый.ДополнитьМассивОписанияВозможныхКонтактов

**Comment by capitoshko** [comment]
Все доработки делать там?

**Comment by pallid** [comment]
Где там?

**Comment by pallid** [comment]
Нет, конкретно эту доработку лучше так вынести, тем самым будет отделен код БСП от кода конфигурации tasks

и модуль ВзаимодействияКлиентСервер можно будет вернуть на поддержку

**Comment by pallid** [comment]
просто при беглом взгляде при сравнении tasks и БСП это что первое на глаза попалось, надо дальше еще смотреть что надо бы вынести в свои модули перед выполнением #117

**Comment by capitoshko** [comment]
Я в целом согласен 


Там - модуль для системы задач

**Comment by pallid** [comment]
Это про узОбщийМодульСервер или узОбщийМодульКлиент? Но надо тогда еще КлиентСервер иначе придется это писать в двух модулях и вызывать в зависимости от контекста приложения

**Comment by pallid** [comment]
Вообще надо позвать @BlizD и поинтересоваться стоит ли это или зря заморачиваемся

**Comment by pallid** [comment]
> Я писал о модулях взаимодействия

я видимо не понял, можно по конкретней

**Comment by BlizD** [comment]
Думаю, пока не стоит с этим заморачиваться, код выделен "обрамлен", если надо будет вынести его в наши модули, тогда сделаем.

**Comment by pallid** [comment]
> код выделен "обрамлен

кстати, зачем обрамлять, проект же версионируется?

**Comment by BlizD** [comment]
@pallid 
>кстати, зачем обрамлять, проект же версионируется?

В объектах, которые относятся к БСП, все же пока обрамляем, чтобы бы было легче обновить БСП.
Что касается других мест, то если изменение проходит через pull request тогда комментарии не нужны. Но часть изменений приходит через просто CF файлы, поэтому там обрамление остается.

**Comment by pallid** [comment]
ну у бсп за частую есть переопределяемые модули для этого
  
>Среда, 24 октября 2018, 14:13 +03:00 от Anton Ivanov <notifications@github.com>:
> 
>@pallid
>>кстати, зачем обрамлять, проект же версионируется?
>В объектах, которые относятся к БСП, все же пока обрамляем, чтобы бы было легче обновить БСП.
>Что касается других мест, то если изменение проходит через pull request тогда комментарии не нужны. Но часть изменений приходит через просто CF файлы, поэтому там обрамление остается.
>—
>You are receiving this because you were mentioned.
>Reply to this email directly,  view it on GitHub , or  mute the thread . 
 
 
--
с уважением,
Василий Попов
